File: divisibleSumPairs_hacker_rank.rb
def divisibleSumPairs(n, k, ar)
     count = 0
    (0...ar.length - 1).each do |i|
        (i + 1...ar.length).each do |j|
            count += 1 if (ar[i] + ar[j]) % k == 0
        end
    end
    count

end

File: divisibleSumPairs_hackerrank.js
function divisibleSumPairs(n, k, ar) {
    let count = 0;
    for (let i = 0; i < ar.length - 1; i++) {
        let num1 = ar[i];
        for (let j = i + 1; j < ar.length; j++) {
            let num2 = ar[j];
            if ((num1 + num2) % k == 0) {
                count += 1;
            }
        }
    }
    return count;
}
